deploy/frontend.yaml

```diff
@@ -175,6 +188,10 @@ objects:
                   permissions:
                     - method: isOrgAdmin
                   product: Identity & Access Management
+        - segmentId: rbac-learning-resources
```

Comment:
We might be able to remove this since its also defined here https://github.com/RedHatInsights/learning-resources/blob/master/deploy/frontend.yml#L52